remind_me/static/js/addrem.js

```diff
@@ -0,0 +1,166 @@
+    
+function showreminder(){
+    // var token = '2200d7faf10f34cd9df0732fe49246560f282a4a';
+    ihtml = "";
+    jhtml = "";
+     $.ajax({
+        url: "/reminders/",
+        type: "GET",
+        headers: {
+            'Authorization': 'Token '+token
+        },
+        success : function(data){
+            values=data;
+            var ihtml="";
+            for(var i=0;i<data.length;i++){
+                msg = data[i].message;
+                date = data[i].scheduled_datetime;
+                id=data[i].id;
+                phone = data[i].phone;
+                // ihtml_temp = createreminder(msg,date,id,phone,i);
+                // ihtml += ihtml_temp;
+                createreminder(msg,date,id,phone,i);
+            }
+            // document.getElementById("listreminder").innerHTML = ihtml;
+        }
+    });
+   
+}
+
+
+
+function createreminder(msg,fulldate,id,phone,i){
+    var newDate = new Date();
+    todayYear = newDate.getFullYear();
+    todayMonth = newDate.getMonth()
+    todayDate =  newDate.getDate() ;
+    hour = newDate.getHours();
+    min = newDate.getMinutes();
+    sec = newDate.getSeconds();
+    ms = newDate.getMilliseconds();
+    var today = new Date(todayYear,todayMonth,todayDate,hour,min,sec,ms);
+    
+    //console.log(datetime);
+    var datefull = datesplit(fulldate);
+    // datefull = datefull.toDateTimeString();
+    if(datefull >= today ){
+        
+        ihtml += '<li class="ui remind"><input name="idinp" type="hidden" value="'+id+'">'+'<p class="messagetext">'+msg+'</p>'+'<label class="ui label datelabel">' + datefull.toDateString() + ' at '+ datefull.toTimeString() +'</label> <div class="ui right floated red button deletebtn" onclick="deletereminder('+i+')"> Delete </div> <div class="ui teal button right floated editbtn" onclick="editreminder('+i+')"> Edit </div>  </li>' ;
+    }
+    else{
+        
+        jhtml += '<li class="ui pastremind"><input name="idinp" type="hidden" value="'+id+'">'+'<p class="pastmessagetext">'+msg+'</p>'+'<label class="ui label datelabel">'+  datefull.toDateString() + ' at '+ datefull.toTimeString() +'</label> <div class="ui right floated red button deletebtn" onclick="deletereminder('+i+')"> Delete </div> <div class="ui blue button right floated editbtn" onclick="editreminder('+i+')"> Remind Again </div>  </li>' ;
+    }  
+    document.getElementById("listreminder").innerHTML = ihtml;
+    document.getElementById("listpastreminder").innerHTML = jhtml;
+}
+function editreminder(i){
+    var item = document.getElementById("listreminder").children[i];
+    str = "list-style-type";
+    item.style.str   = "none";
+    var id = item.children[0].value;
+    var result = $.grep(values, function(e){ return e.id == id; });
+    var date=datesplit(result[0].scheduled_datetime);
+    var itemhtml = "" ;
+    itemhtml = '<div class="ui input msgdiv" > <input id="newmsg" class="message" type="text" name="message" value ="'+ result[0].message+'"     > </div> ';
```

Comment:
@harshamotwani22 You could have used any simple template engine like underscore, handlebars. Instead of manipulation the html like this. Are you familiar with any template engine?